website/docs/configuration.md

```diff
@@ -34,22 +34,76 @@ Deployment configurations such as `projectName` and `organizationName` are used
 
 It is recommended to check the [deployment docs](deployment.md) for more information.
 
-### Themes, Plugins, and Presets configurations
+### Theme, plugin, and preset configurations
 
-_This section is a work in progress. [Welcoming PRs](https://github.com/facebook/docusaurus/issues/1640)._
+List the installed [themes](using-themes.md), [plugins](using-plugins.md), and [presets](presets.md) for your site in the `themes`, `plugins`, and `presets` fields, respectively. These are typically npm packages:
 
-<!--
+```js
+// docusaurus.config.js
+module.exports = {
+	// ...
```

Comment:
Here and below please use spaces only, not tabs.

Reply:
Good to know, thank you!